prog/types.go

```diff
@@ -596,8 +607,8 @@ func (t *PtrType) isDefaultArg(arg Arg) bool {
 
 type StructType struct {
 	TypeCommon
-	Fields    []Field
-	AlignAttr uint64
+	Fields     []Field
+	PackedAttr bool
```

Comment:
This should not be needed if we have TypeCommon.TypeAlignment, right?

Reply:
Correct, removed the packed attr.